README.md

```diff
@@ -13,22 +14,70 @@ Running full scikit-learn test suite with daal4p's optimization patches
 - [![CircleCI](https://circleci.com/gh/IntelPython/daal4py.svg?style=svg)](https://circleci.com/gh/IntelPython/daal4py) when applied to scikit-learn from PyPi
 - [![CircleCI](https://circleci.com/gh/IntelPython/daal4py/tree/test-sklearn-master.svg?style=svg)](https://circleci.com/gh/IntelPython/daal4py/tree/test-sklearn-master) when applied to build from master branch
 
-With this daal4py API, your Python programs can use Intel(R) oneAPI Data Analytics Library algorithms in just one line:
+Core functioanlity of daal4py is in place Scikit-learn patching - Same Code, Same Behavior but faster execution. 
+
+Stock Scikit-learn
+```
+from sklearn.svm import SVC
+X, Y = get_dataset()
```

Comment:
Probably, it's better to replace this with:
```py
from sklearn.datasets import load_digits
digits = load_digits()
X, y = digits.data, digits.target
```
The same how we do it in oneDAL's README to allow to run the code by copy-past only.

Reply:
done